src/hotspot/share/jfr/recorder/checkpoint/jfrCheckpointManager.cpp

```diff
@@ -91,8 +91,8 @@ JfrCheckpointManager::~JfrCheckpointManager() {
 static const size_t global_buffer_prealloc_count = 2;
 static const size_t global_buffer_size = 512 * K;
 
-static const size_t thread_local_buffer_prealloc_count = 16;
-static const size_t thread_local_buffer_size = 128;
+static const size_t thread_local_buffer_prealloc_count = 32;
```

Comment:
Why is more memory needed? Moore's law or something specific to virtual threads?